content/handbook/product-development/how-we-work/issue-triage.md

```
@@ -0,0 +1,437 @@
+これらの Issue は、[確立されたバグの severity SLO](/handbook/product-development/how-we-work/issue-triage/#severity-slos) に従います。
+
+## Issue のトリアージ
+
+初期トリアージでは、（少なくとも）Issue に適切なラベルを付けます。これにより、トリアージされていない Issue はラベルのない Issue を検索することで見つけられます。
+
+次のいずれかのリンクをたどってください：
+
+- [GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=None&assignee_id=None)
+- [GitLab Omnibus](https://gitlab.com/gitlab-org/omnibus-gitlab/-/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=None&assignee_id=None)
+
+Issue を 1 つ選び（リスト内で最も古いものを優先）、下の [Issue トリアージの慣行](#issue-triage-practices) を念頭に置いて、批判的な目で評価してください。自問するべきいくつかの質問：
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)